content/posts/cicd_ha.md

```diff
@@ -0,0 +1,132 @@
+---
+authors:
+  - "David Cañadillas"
+title: "CI/CD High Availability: Misunderstanding the concept"
```

Comment:
This content is excellent @dcanadillas ! If the title was tweaked to something slightly more positive like "CI/CD High Availability: What's truly needed" (this is terrible but you get the idea...) - - I feel this could be used immediately with a couple of high profile customers in the US who are very stuck in their definitions so we need to invite them nicely to conversation. :-)

Reply:
Regarding the title, I kinda get @aprilkmace 's point - it comes across a bit like people aren't smart enough to understand it. 

Another suggestion: "CD High Availability: It's What You Really Need, Not What You Think You Need"

Of course if you want to be really funny it could be "CD High Availability: You Can't Always Get What You Want"

Regarding dropping CI - you can't do CD without CI, and we want our customers doing CD, right?

Reply:
One more title idea: A Practical Approach to High Availability for Continuous Delivery

Reply:
I see the point of removing CI part and I agree about being implicit in the CD definition. But IMHO wouldn't remove it because I would think about those that are still doing only CI but on the path for CD (a lot of Jenkins users are in that stage).

About the title. I just changed it for "CI/CD High Availability: The Real Thing", just trying to get it about all the good stuff from your reviews. What do you think??

Reply:
@dcanadillas I wouldn't use "the real thing" just because it was a Coca Cola slogan for a long time and people will be looking for a connection that isn't there. My preference so far (for what it's worth) is @kmadel 's suggestion "A Practical Approach to CI/CD High Availability" - maybe you could still do something punchy before it - still having trouble with the right words but I haven't had caffeine yet. :-) Any ideas here?

Reply:
@aprilkmace A little late, as it has already been published. But I guess we could republish it. @dcanadillas what do you think?